app/Services/DebugRequestActivityLogger.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Services;
+
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+
+class DebugRequestActivityLogger extends AbstractRequestActivityLogger
+{
+    protected function collectRequestData(Request $request): array
+    {
+        return [
+            "Method: " . $request->method() . ". " . "Url: " . $request->fullUrl() . ". " . "Time: " . Carbon::now() . ". "
```

Comment:
you have array, please use associative array to organize it, instead of single string.

Reply:
Same for ProductionRequestActivityLogger

Reply:
Modified Debug and Production request activity loggers to use associative array instead of single string